src/panels/profile/ha-pick-theme-row.ts

```diff
@@ -0,0 +1,240 @@
+                @change=${this._handleDarkMode}
+                name="dark_mode"
+                value="dark"
+                ?checked=${this.hass.selectedTheme?.dark === true}
+              >
+              </ha-radio>
+            </ha-formfield>
+            <div class="color-pickers">
+              <paper-input
+                .value=${this.hass!.selectedTheme?.primaryColor || "#03a9f4"}
+                type="color"
+                .label=${this.hass!.localize(
+                  "ui.panel.profile.themes.primary_color"
+                )}
+                .name=${"primaryColor"}
+                @change=${this._handleColorChange}
+              ></paper-input>
+              <paper-input
+                .value=${this.hass!.selectedTheme?.accentColor || "#ff9800"}
+                type="color"
+                .label=${this.hass!.localize(
+                  "ui.panel.profile.themes.accent_color"
+                )}
+                .name=${"accentColor"}
+                @change=${this._handleColorChange}
+              ></paper-input>
+              ${this.hass!.selectedTheme?.primaryColor ||
+              this.hass!.selectedTheme?.accentColor
+                ? html` <mwc-button @click=${this._resetColors}>
+                    ${this.hass!.localize("ui.panel.profile.themes.reset")}
+                  </mwc-button>`
+                : ""}
+            </div>
+          </div>`
+        : ""}
+    `;
+  }
+
+  protected updated(changedProperties: PropertyValues) {
+    const oldHass = changedProperties.get("hass") as undefined | HomeAssistant;
+    const themesChanged =
+      changedProperties.has("hass") &&
+      (!oldHass || oldHass.themes?.themes !== this.hass.themes?.themes);
+    const selectedThemeChanged =
+      changedProperties.has("hass") &&
+      (!oldHass || oldHass.selectedTheme !== this.hass.selectedTheme);
+
+    if (themesChanged) {
+      this._themes = ["Backend-selected", "default"].concat(
+        Object.keys(this.hass.themes.themes).sort()
+      );
+    }
+
+    if (selectedThemeChanged) {
+      if (
+        this.hass.selectedTheme &&
+        this._themes.indexOf(this.hass.selectedTheme.theme) > 0
+      ) {
+        this._selectedTheme = this._themes.indexOf(
+          this.hass.selectedTheme.theme
+        );
+      } else if (!this.hass.selectedTheme) {
+        this._selectedTheme = 0;
+      }
+    }
+  }
+
+  private _handleColorChange(ev: CustomEvent) {
+    const target = ev.target as any;
+    fireEvent(this, "settheme", { [target.name]: target.value });
+  }
+
+  private _resetColors() {
+    fireEvent(this, "settheme", {
+      primaryColor: undefined,
+      accentColor: undefined,
+    });
+  }
+
+  private _handleDarkMode(ev: CustomEvent) {
+    let dark: boolean | undefined;
+    switch ((ev.target as HaRadio).value) {
+      case "light":
+        dark = false;
+        break;
+      case "dark":
+        dark = true;
+        break;
+    }
+    fireEvent(this, "settheme", { dark });
```

Comment:
So one thing that we're going to get feedback on is that people will want to share their theme with other users. I guess we can facilitate that in a future PR (probably needs support in frontend integration)

Reply:
It is just 2 colors?

Reply:
Right, but the other people in my house won't take the time to configure a theme, but maybe my Lovelace setup requires that. I guess we can do that in backend selected still 🤷

Reply:
Oh, you mean users in your HA, I was thinking users as the community. I understand it now :-)

Reply:
I think we could add these vars to the set theme service call, and add a button in the frontend to set the current settings as default.

We should make sure these settings are stored on the backend btw, as it now doesn't survive a restart?